website/templates/website/contact.html

```diff
@@ -1,17 +1,17 @@
 {% extends 'website/base.html' %}
 {% block content %}
 {% load crispy_forms_tags %}
-
-<div style="content-fluid; margin-top: 200px; margin-bottom: 160px">
-    <div class="row" style="padding-top: 100px; padding-bottom: 100px">
+<img src="/media/images/phone_.jpg" style="width:100%">
+<div style="content-fluid; margin-top: 50px; margin-bottom: 50px">
+    <div class="row" style="padding-left: 100px; padding-right: 100px; padding-top: 50px; padding-bottom: 50px">
```

Comment:
consider moving some of these styles to a stylesheet

Reply:
I will do it later altogether.